src/NuGet.Core/NuGet.Protocol/Converters/NuGetFrameworkConverter.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Xml.Linq;
+using Newtonsoft.Json;
+using NuGet.Frameworks;
+
+namespace NuGet.Protocol.Converters
+{
+    public class NuGetFrameworkConverter : JsonConverter
```

Comment:
```suggestion
    internal class NuGetFrameworkConverter : JsonConverter
```

I don't think there's any good reason we need to make this public: https://github.com/NuGet/NuGet.Client/blob/dev/docs/nuget-sdk.md#changes-to-apis